src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1032,6 +1032,11 @@ public async Task LoadExtendedItemPropertiesAsync(ListedItem item)
 							BaseStorageFolder matchingStorageFolder = await GetFolderFromPathAsync(item.ItemPath, cts.Token);
 							if (matchingStorageFolder is not null)
 							{
+								cts.Token.ThrowIfCancellationRequested();
+								await LoadThumbnailAsync(item);
+
+								cts.Token.ThrowIfCancellationRequested();
```

Comment:
```suggestion
								_ = LoadThumbnailAsync(item);
```
This seems to avoid crashes.